docs/11_streamline_workflow_with_aliases.md

```diff
@@ -44,26 +44,20 @@ Pro Tip # 1:  To edit aliases by hand, you can open the gitconfig file with your
 git config --global alias.ec "config --global -e"
 ```
 
-Pro Tip # 2: To checkout to another branch, you can make a quick shortcut.
+Pro Tip # 2: To switch to another branch, you can make a quick shortcut.
 
 ```sh
-git config --global alias.ch "checkout"
+git config --global alias.sw "switch"
 ```
 
-Pro Tip # 3: To checkout to a brand new branch, you can easily extend your existing shortcut.
+Pro Tip # 3: To create and switch to a brand new branch, you can easily extend your existing shortcut.
 
 ```sh
-git config --global alias.cob "checkout -b"
+git config --global alias.swc "switch -c"
 ```
 
 Pro Tip # 4: You can create aliases that only call one command.
 
 ```sh
 git config --global alias.s "status -s"
 ```
-
-Pro Tip # 5: Clean up branches quickly and easily.
-
-```sh
-git config alias.dlb '!git checkout <DEFAULT-BRANCH> && git pull --prune && git branch --merged | grep -v "\*" | xargs -n 1 git branch -d'
```

Comment:
This doesn't work for me - maybe someone can test this out

Reply:
@rwnfoo did you replace `<DEFAULT-BRANCH>` with the name of the default branch in the repo you ran this in?

Reply:
I’ve opened a PR to restore this pro tip: https://github.com/githubtraining/training-manual/pull/297